canonical/microk8s.io
# Apply redesign /tutorials

## Done

- Upgraded site to latest Vanilla version
- Applied redesign as per [doc](https://docs.google.com/document/d/1Lpo7Od9VA1aHYmr9UVp02vCGOP5Zxmzx4zZE9YJwlQg/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?type=design&node-id=94%3A4183&mode=design&t=04EfFbvHoCVgWIP2-1)

## QA

- View the site locally in your web browser at: https://microk8s-io-606.demos.haus/tutorials
- Check page against doc and design


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4188
